### PR TITLE
Added controllers for Movie, MovieSession, User, and DTOs for them

### PR DESCRIPTION
**Pull request by vladkuzya**

### CODE REVIEW

**Review by Borisko130 — APPROVED** [review]

File: src/main/java/mate/academy/config/AppConfig.java [hunk 1]
```diff
@@ -15,7 +15,8 @@
 @ComponentScan(basePackages = {
         "mate.academy.service",
         "mate.academy.dao",
-        "mate.academy.security"
+        "mate.academy.security",
+        "mate.academy.model.dto"
```
Comment (Sofasmile) [inline src/main/java/mate/academy/config/AppConfig.java]: should we use dto here?

File: src/main/java/mate/academy/model/dto/response/MovieResponseDto.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package mate.academy.model.dto.response;
+
+public class MovieResponseDto {
+    private Long id;
+    private String title;
```
Comment (Sofasmile) [inline src/main/java/mate/academy/model/dto/response/MovieResponseDto.java:5]: we can return description too))

File: src/main/java/mate/academy/model/dto/response/MovieSessionResponseDto.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
+package mate.academy.model.dto.response;
+
+public class MovieSessionResponseDto {
+    private Long id;
+    private String movieTitle;
```
Comment (Sofasmile) [inline src/main/java/mate/academy/model/dto/response/MovieSessionResponseDto.java:6]: let's add movie id too

File: src/main/java/mate/academy/service/mapper/UserMapper.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package mate.academy.service.mapper;
+
+import mate.academy.model.User;
+import mate.academy.model.dto.request.UserRequestDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserMapper {
```
Comment (Sofasmile) [inline src/main/java/mate/academy/service/mapper/UserMapper.java]: - do we use UserMapper ? let's remove this class

File: src/main/java/mate/academy/config/AppConfig.java (hunk 1 above)
Comment (vladkuzya) [reply src/main/java/mate/academy/config/AppConfig.java]: No, we shouldn't)

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**